channels.go

```diff
@@ -0,0 +1,105 @@
+package mackerel
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// Channel represents a Mackerel notification channel.
+// ref. https://mackerel.io/api-docs/entry/channels
+type Channel struct {
+	ID string `json:"id"`
+	ChannelWithoutID
+}
+
+// ChannelWithoutID represents a Mackerel notification channel without the ID.
+type ChannelWithoutID struct {
+	Name string `json:"name"`
+	Type string `json:"type"`
+
+	// Exists when the type is "email"
+	Emails  []string `json:"emails,omitempty"`
+	UserIDs []string `json:"userIds,omitempty"`
+
+	// Exists when the type is "slack"
+	Mentions          Mentions `json:"mentions,omitempty"`
+	EnabledGraphImage bool     `json:"enabledGraphImage,omitempty"`
+
+	// Exists when the type is "slack" or "webhook"
+	URL string `json:"url,omitempty"`
+
+	// Exists when the type is "email", "slack", or "webhook"
+	Events []string `json:"events,omitempty"`
+}
+
+// Mentions represents the structure used for slack channel mentions
+type Mentions struct {
+	OK       string `json:"ok,omitempty"`
+	Warning  string `json:"warning,omitempty"`
+	Critical string `json:"critical,omitempty"`
+}
+
+// ListChannels requests the channels API and returns a list of Channel
+func (c *Client) ListChannels() ([]*Channel, error) {
```

Comment:
In mackerel-client-go, almost GET-ish method names is prefixed *Find*, except *ListHostMetricNames* and *ListServiceMetricNames*. Would you consider whether to rename?

Reply:
Ah. Personally, I feel `Find...` suggests that it only returns 1 item, and `List...` returns an array of items. However, it is better to be consistent so I will accept your suggestion. Thanks!